src/java/org/apache/cassandra/db/compaction/ShardManager.java

```diff
@@ -83,6 +89,12 @@ else if (partitioner.splitter().isPresent())
      */
     double shardSetCoverage();
 
+    /**
+     * The minimum token space share per partition that should be assigned to sstables with small numbers of partitions
+     * or which have fallen outside the local token ranges.
+     */
+    double minimumPerPartitionSpan();
```

Comment:
We should merge or rebase on `main` because the `ShardManagerReplicaAware` needs to implement this method.

Reply:
Done